qa/rustfmt.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/sh
+
+r=0
+# check the list of known rustfmt-ed files
+cat qa/rustfmt.txt | while read i; do
+    cargo fmt $i;
+    # the file used to be formatted and is not anymore
+    if [ $(git diff -- $i | wc -l) -gt 0 ]; then
+        git diff;
```

Comment:
This will print diffs of all files, not just the current file.

Probably meant `git diff -- $i`?

Reply:
But `cargo fmt $i` only changed one file, right ?

Reply:
Hmm, looking at that at https://github.com/rust-lang/rustfmt#running

Seems `cargo fmt` does the whole crate/module. It might silently ignore the file?

You'd have to use `rustfmt` it seems with files. However, `rustfmt` might also check some included files if I got the description correct?! So that might check transiently included files multiple times? Ugh, that would open up "interesting behaviours"... Maybe worth trying it out?

If that is the case, it might be easier to run `cargo fmt` once and then `git diff` check for the list of files in `rustfmt.txt`?

Reply:
Thanks Roland for the tips, waiting for a decision wether this is the right approach before tuning this